Index: src/main/java/com/autobots/new_version.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/autobots/new_version.java b/src/main/java/com/autobots/new_version.java
new file mode 100644
--- /dev/null	(date 1751864514545)
+++ b/src/main/java/com/autobots/new_version.java	(date 1751864514545)
@@ -0,0 +1,4 @@
+package com.autobots;
+
+public class new_version {
+}
